tests/pom.xml

```diff
@@ -592,7 +592,7 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
                     <env>
                       <LOGGING_CONFIG>file:/etc/hono/logback-spring.xml</LOGGING_CONFIG>
                       <SPRING_CONFIG_LOCATION>file:/etc/hono/</SPRING_CONFIG_LOCATION>
-                      <SPRING_PROFILES_ACTIVE>prod</SPRING_PROFILES_ACTIVE>
+                      <SPRING_PROFILES_ACTIVE>trace</SPRING_PROFILES_ACTIVE>
```

Comment:
this should not be committed to the repo FMPOV.

Reply:
Sure, that was just an extra commit to have more travis log output. That commit is to be removed before the merge.